test/Microsoft.ML.Functional.Tests/Microsoft.ML.Functional.Tests.csproj

```diff
@@ -0,0 +1,53 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AssemblyName>Microsoft.ML.Functional.Tests</AssemblyName>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <!-- We are turning off strong naming to ensure we never add `InternalsVisibleTo` for these tests -->
+    <SignAssembly>false</SignAssembly>
```

Comment:
You probably need to set `PublicSign` to false as well, to unblock the Mac and Linux builds.

Reply:
Thanks for the tip! I couldn't figure out why those builds broke!

---
In reply to: [255168732](https://github.com/dotnet/machinelearning/pull/2470#discussion_r255168732) [](ancestors = 255168732)